src/main/java/bsh/Name.java

```diff
@@ -1020,8 +1020,12 @@ public String suffix(int parts) {
             return suffix[parts];
         }
         public static Parts get(String value) {
-            if (PARTSCACHE.containsKey(value))
-                return PARTSCACHE.get(value);
+            if (PARTSCACHE.containsKey(value)) {
+                Parts parts = PARTSCACHE.get(value);
+                if (null != parts)
+                    return parts;
+                PARTSCACHE.remove(value);
```

Comment:
But it is green Codecov so what gives? It is tested because this change actually fixes an existing test failing.